bin/build-scripts/ftl-to-js.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+
+// This is called by make. argv[2] is the destination directory. The the args
+// after are the paths to ftl files.
+//
+// The destination directory has been created at this point.
+
+const fs = require('fs');
+const destDir = process.argv[2];
+const ftlPaths = process.argv.slice(3);
+
+for (let ftl of ftlPaths) {
+  let locale = ftl.split('/')[1];
+  let destFilePath = `${destDir}/${locale}.js`;
+  let ftlFileContent = fs.readFileSync(ftl, 'utf8');
+  let jsFileContent = `
+    (context => {
+      let messages=${JSON.stringify(ftlFileContent)};
+      if (typeof exports === 'object' && context === exports) {
+        exports.messages = messages;
+      } else {
+        context.l10nMessages = context.l10nMessages || {};
+        context.l10nMessages['${locale}'] = messages;
+      }
```

Comment:
This would be a good place to put in a call to the effect of "your l10n messages have arrived".  E.g., `if (typeof window !== "undefined" && window.notifyL10nLoaded) {window.notifyL10nLoaded(locale);}`